### ISSUE TITLE
Where can we use json schema?

### ISSUE THREAD

**Issue by agentlewis** [enspiral-root-systems/dogstack#58]
I have been thinking about this in regards to how to start projects. I am very excited about the idea of being able to use json schema as a first pass for several things:

 - define the data shape => would become the database.
 - define the api (if needed) => would often be used for public access or to use as a contract between a backend and front end team.
 - define the data required for specific events => would often become a query
 - define the data required for specific views => would often become a getter

In other words I want to start thinking of different parts of an app based upon what data is being riceived or sent at that place.

I see this as supplementary to getting into writing gherkin stories.
